lib/matplotlib/path.py

```diff
@@ -95,6 +95,9 @@ class Path:
                              CURVE4: 3,
                              CLOSEPOLY: 1}
 
+    # A list of path codes for which the vertices are ignored
+    CODES_WITH_IGNORED_VERTICES = [STOP, CLOSEPOLY]
```

Comment:
Does this need to be public? Then it should probably have a docstring (`#:`).

Reply:
It doesn't need to be, but it might supplement the main docstring above to have this public and documented.

Reply:
Ah, I've removed it as per the comment below